src/vivarium_public_health/population/add_new_birth_cohorts.py

```diff
@@ -158,36 +156,38 @@ def setup(self, builder):
         builder : vivarium.engine.Builder
             Framework coordination object.
         """
+        age_specific_fertility_rate = self.load_age_specific_fertility_rate_data(builder)
```

Comment:
This whole component needs serious attention before we put it in a production model.  I wrote it when I didn't understand demography/our sim conventions super well and it will be mildly incorrect and confusing to use.